pkg/scheduler/plugins/reservation/plugin.go

```diff
@@ -236,6 +236,24 @@ func (p *Plugin) PostFilter(ctx context.Context, state *framework.CycleState, po
 		// return err to stop default preemption
 		return nil, framework.NewStatus(framework.Error)
 	}
+
+	nodeinfos, err := p.handle.SnapshotSharedLister().NodeInfos().List()
+	if err != nil {
+		return nil, framework.NewStatus(framework.Error)
+	}
+
+	p.parallelizeUntil(ctx, len(nodeinfos), func(piece int) {
+		nodeInfo := nodeinfos[piece]
+		for _, pi := range nodeInfo.Pods {
+			if util.IsReservePod(pi.Pod) {
+				maxPri := int32(math.MaxInt32)
+				newReservePod := pi.Pod.DeepCopy()
+				newReservePod.Spec.Priority = &maxPri
```

Comment:
Can only be processed when the reservePod's priority is lower than the Pod's priority.